application/models/Logbook_model.php

```diff
@@ -3727,7 +3727,7 @@ function qrz_update($primarykey, $qsl_date, $qsl_status) {
 		}
 	}
 
-	function lotw_update($datetime, $callsign, $band, $qsl_date, $qsl_status, $state, $qsl_gridsquare, $qsl_vucc_grids, $iota, $cnty, $cqz, $ituz, $station_callsign, $qsoid, $station_ids, $dxcc = null, $ant_path = null) {
+	function lotw_update($datetime, $callsign, $band, $qsl_date, $qsl_status, $state, $qsl_gridsquare, $qsl_vucc_grids, $iota, $cnty, $cqz, $ituz, $station_callsign, $qsoid, $station_ids, $country = null, $dxcc = null, $ant_path = null) {
```

Comment:
```suggestion
	function lotw_update($datetime, $callsign, $band, $qsl_date, $qsl_status, $state, $qsl_gridsquare, $qsl_vucc_grids, $iota, $cnty, $cqz, $ituz, $station_callsign, $qsoid, $station_ids, $dxcc = null, $country = null, $ant_path = null) {
```

I think you reversed country and dxcc here ...

Reply:
This even throws a mysql error:

![image](https://github.com/user-attachments/assets/fc49c690-bdb7-4909-bcd5-22b1476fc3a5)